runtime/type_conversions_unittests.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// FLUTTER_NOLINT
+
+#include "flutter/runtime/dart_vm_lifecycle.h"
+#include "flutter/testing/dart_isolate_runner.h"
+#include "flutter/testing/fixture_test.h"
+#include "flutter/testing/testing.h"
+#include "flutter/third_party/tonic/converter/dart_converter.h"
+
+namespace flutter {
+namespace testing {
+
+class TypeConversionsTest : public FixtureTest {
+ public:
+  TypeConversionsTest()
+      : settings_(CreateSettingsForFixture()),
+        vm_(DartVMRef::Create(settings_)) {}
+
+  ~TypeConversionsTest() = default;
+
+  [[nodiscard]] bool RunWithEntrypoint(const std::string& entrypoint) {
+    if (running_isolate_) {
+      return false;
+    }
+    auto thread = CreateNewThread();
+    TaskRunners single_threaded_task_runner(GetCurrentTestName(), thread,
+                                            thread, thread, thread);
+    auto isolate =
+        RunDartCodeInIsolate(vm_, settings_, single_threaded_task_runner,
+                             entrypoint, {}, GetFixturesPath());
+    if (!isolate || isolate->get()->GetPhase() != DartIsolate::Phase::Running) {
+      return false;
+    }
+
+    running_isolate_ = std::move(isolate);
+    return true;
+  }
+
+ private:
+  Settings settings_;
+  DartVMRef vm_;
+  std::unique_ptr<AutoIsolateShutdown> running_isolate_;
+  FML_DISALLOW_COPY_AND_ASSIGN(TypeConversionsTest);
+};
+
+TEST_F(TypeConversionsTest, TestFixture) {
+  ASSERT_TRUE(RunWithEntrypoint("main"));
+}
+
+// TODO(WIP): Add test for additional entries in dart_convertor.
```

Comment:
Should that go in this PR, or should we file an issue for it?

Reply:
Oops. I thought I cleaned up all `TODO(WIP)`s. No, I'll add the tests as there are no tests in Tonic for the convertors I touched.

Reply:
Done. [Added the tests](https://github.com/flutter/engine/pull/21820/commits/eb4f87627f51959ce5491d2cb1db5f3276e33ca7) meant for this harness. Also checked again to make sure I didn't miss any WIP items.